src/algebra/group_power/lemmas.lean

```diff
@@ -800,13 +786,13 @@ h.units_zpow_right m
 
 variables {a b : R}
 
-@[simp] lemma cast_int_mul_right (h : commute a b) (m : ℤ) : commute a (m * b) :=
+@[simp] lemma cast_int_mul_right (h : commute a b) (m : ℤ) : commute a (m * b : R) :=
```

Comment:
Note: I have no idea why this broke. Somehow Lean doesn't figure out the type of `m * b` from the type of `a`.